README.md

```diff
@@ -57,6 +57,17 @@ In the end, run the test suite to see if all went well
 make test
 ```
 
+If you want to use the benchmarks, checkout google benchmark from github, build and 
```

Comment:
This should be part of the developer documentation in Doxygen, rather than in the README, imho.

Reply:
Okay, will be removed with next commit.